core/backend/src/AliCloudStorageService.ts

```diff
@@ -36,7 +41,10 @@ declare namespace OSS { // eslint-disable-line no-redeclare
   }
 }
 
-/** @beta */
+/**
+ * @beta
+ * @deprecated Use `@itwin/object-storage-oss` instead, see https://github.com/iTwin/object-storage for more details.
```

Comment:
We should work to have the object-storage packages represented in the itwinjs.org docs site. I'll make a note of that and hopefully we can resolve it within the next few weeks.